libvast/src/system/application.cpp

```diff
@@ -97,7 +97,7 @@ auto make_root_command(std::string_view path) {
         .add<std::string>("config", "path to a configuration file")
         .add<caf::atom_value>("verbosity", "output verbosity level on the "
                                            "console")
-        .add<std::vector<std::string>>("schema-paths", schema_desc.c_str())
+        .add<std::vector<std::string>>("schema-dirs", schema_desc.c_str())
```

Comment:
The changelog entry says that the old options are deprecated, but this change removes them completely from the cli and they will only work from `vast.yaml`. I think we should strive to keep both in sync.